packages/contracts/deploy/009-Proxy__OVM_L1StandardBridge.deploy.ts

```diff
@@ -21,14 +17,31 @@ const deployFn: DeployFunction = async (hre) => {
     }
   )
 
-  const result = await deploy('Proxy__L1StandardBridge', {
+  // Deploy the Proxy
+  const resultProxyDeploy = await deploy('Proxy__L1StandardBridge', {
```

Comment:
We'll be keeping the existing chugsplash proxy, so I think we'll need to provide an option here for reusing an existing one. 

Alternatively we might be able to help @smartcontracts get the [ChugSplash package](https://github.com/ethereum-optimism/optimism/tree/regenesis/0.5.0/packages/chugsplash) ready for this